packages/storage-adapter-local-forage/src/index.js

```diff
@@ -56,16 +55,9 @@ export default class StorageAdapterLocalForage {
        */
       get(key) {
         const key_ = `${namespaces.get(this)}/${key}`;
-        loggers.get(this).info(
-          `local-forage-store-adapter: reading \`${key_}\``);
+        loggers.get(this).info(`local-forage-store-adapter: reading \`${key_}\``);
         return localforage.getItem(key_)
-          .then((value) => {
-            if (value) {
-              return Promise.resolve(value);
-            }
-            return Promise.reject(new NotFoundError(
-              `No value found for ${key_}`));
-          });
+          .then((value) => value || Promise.reject(new NotFoundError(`No value found for ${key_}`)));
```

Comment:
Actually, i gave you bad advice (and there may be a bug in the local storage adapter as well). This setup will prevent us from storing `0` or `false`. 

Can you add a test that writes and reads `false` from the db and also take a look if local storage and memory store adapters have the same bug?